src/plugtest/plugtest-server.c

```diff
@@ -90,7 +90,7 @@ plugtest_test_handler(nyoci_node_t node)
 			strlcat(content,coap_option_key_to_cstr(key,1),max_len);
 			strlcat(content,": ",max_len);
 			if(coap_option_value_is_string(key)) {
-				int argh = strlen(content)+value_len;
+				coap_size_t argh = strlen(content)+value_len;
```

Comment:
I'm surprised that this didn't itself introduce a warning. `strlen()` returns a `size_t`, which (if I remember correctly) is wider than `coap_size_t` on most platforms. A cast should likely be added.